typings-custom/react-highlight.d.ts

```diff
@@ -1,8 +1,8 @@
 declare module 'react-highlight' {
     import * as __React from 'react';
 
-    interface Highlight extends __React.Component<any, any> {
+    class Highlight extends __React.Component<{ language: string }, {}> {
     }
 
-    export default Highlight;
+    export = Highlight;
```

Comment:
It was the only way TS would stop complaining. :-)  I think this file made its way into the repo as a stray before it was actually used, which is what necessitated the change.